src/overview/tabs/Settings/components/ControllerTransferNetwork/ControllerTransferNetworkHelpContent.tsx

```diff
@@ -0,0 +1,25 @@
+import type { FC } from 'react';
+
+import { Stack, StackItem } from '@patternfly/react-core';
+import { useForkliftTranslation } from '@utils/i18n';
+import { MTV_SETTINGS } from '@utils/links';
+
+const ControllerTransferNetworkHelpContent: FC = () => {
+  const { t } = useForkliftTranslation();
+  return (
+    <Stack hasGutter>
+      <StackItem>
+        {t(
+          'Choose a NetworkAttachmentDefinition for data transfer. If you do not select a NetworkAttachmentDefinition, the default network will be used.',
```

Comment:
Why not detailed that the data transfer is for the forklift controller?
Something like:
`Choose a NetworkAttachmentDefinition for data transfer by the main controller pod.  If you do not select a NetworkAttachmentDefinition, the default network will be used`